hilti/src/rt/backtrace.cc

```diff
@@ -1,38 +1,61 @@
 // Copyright (c) 2020 by the Zeek Project. See LICENSE for details.
 
-#include "rt/backtrace.h"
-
 #include <cstring>
 
 #include <hilti/rt/autogen/config.h>
+#include <hilti/rt/backtrace.h>
 #include <hilti/rt/util.h>
 
 using namespace hilti::rt;
 
 Backtrace::Backtrace() {
 #ifdef HILTI_HAVE_BACKTRACE
-    void* callstack[128];
-    int frames = ::backtrace(callstack, 128);
+    _frames = ::backtrace(_callstack, sizeof(_callstack));
+#endif
+}
 
-    char** strings;
+std::vector<std::string> Backtrace::backtrace() const {
+    std::vector<std::string> bt;
 
-    strings = backtrace_symbols(callstack, frames);
-    assert(strings);
+#ifdef HILTI_HAVE_BACKTRACE
+    assert(_frames >= 0);
+    char** strings = backtrace_symbols(_callstack, _frames);
+    if ( ! strings ) {
+        bt.push_back("# <trouble resolving backtrace symbols>");
```

Comment:
This sounds like something a user could do something about. Could we either tell them what to do (not sure what the error scenarios are), or else show something like `could not resolve backtrace symbols` or similar?

Reply:
seems rare enough to leave as is